tests/conftest.py

```diff
@@ -208,3 +212,221 @@ def mock_sequencing_metrics_get_response(sequencing_metrics_json) -> Mock:
     mock_response.json.return_value = sequencing_metrics_json
     mock_response.raise_for_status.return_value = None
     return mock_response
+
+
+@pytest.fixture
```

Comment:
In CG we consistently use names and scopes for the fixtures, we should do this also here.

```suggestion
@pytest.fixture(name= "", scope=""
```

Reply:
I see the point of setting the scope if it should be something else than the default, but I dislike renaming the fixture with a string in the decorator. It does not seem DRY and is not common practice when using pytest

Reply:
This convention is not applied in this repository so I'm going to leave it since we will most likely also change our guidelines regarding it